common/src/test/java/org/opensearch/ml/common/prompt/MLPromptTest.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.prompt;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.opensearch.common.io.stream.BytesStreamOutput;
+import org.opensearch.common.xcontent.LoggingDeprecationHandler;
+import org.opensearch.common.xcontent.XContentType;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.xcontent.NamedXContentRegistry;
+import org.opensearch.core.xcontent.ToXContent;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.core.xcontent.XContentParser;
+import org.opensearch.ml.common.MLPrompt;
+import org.opensearch.ml.common.TestHelper;
+
+public class MLPromptTest {
+    private MLPrompt mlPrompt;
+
+    private final String expectedInputStr = "{\"prompt_id\":\"dummy promptId\",\"name\":\"some prompt\",\"description\":\"test\","
+        + "\"version\":\"1\",\"prompt\":{\"system\":\"some system prompt\",\"user\":\"some user prompt\"}"
+        + ",\"tags\":[\"test\"],\"tenant_id\":\"tenantId\",\"create_time\":1641600000000,\"last_update_time\":1641600000000}";
+
+    @Before
+    public void setup() {
+        Instant time = Instant.ofEpochSecond(1641600000);
```

Comment:
What is this for? If we are using this only for testing, let's make this more readable like: 

```
Instant time = Instant.parse("2022-01-07T00:00:00Z");
```